bigtable/google/cloud/bigtable/row.py

```diff
@@ -313,20 +311,20 @@ def set_cell(self, column_family_id, column, value, timestamp=None):
                                  Must be of the form
                                  ``[_a-zA-Z0-9][-_.a-zA-Z0-9]*``.
 
-        :type column: bytes
+        :type column: str
```

Comment:
Column qualifiers are likewise bytes, not text, in the [backend docs](https://cloud.google.com/bigtable/docs/reference/data/rpc/google.bigtable.v2#google.bigtable.v2.Column).